cuda_core/cuda/core/experimental/_memory.pyx

```diff
@@ -51,25 +51,27 @@ cdef class Buffer:
     """
 
     cdef:
-        uintptr_t _ptr
+        intptr_t _ptr
```

Comment:
Did this change for any particular reason?

Reply:
I have the same question.

Reply:
Discussed/answered in this thread: https://github.com/NVIDIA/cuda-python/pull/1070#discussion_r2408490288